src/templates/blog-post.js

```diff
@@ -280,6 +283,8 @@ export const pageQuery = graphql`
       fields {
         slug
         langKey
+        modifiedTime: modifiedTime(formatString: "MMMM DD, YYYY")
```

Comment:
gql is weird. the alias come first then the field. 🤦‍♂

Reply:
need to alias ?

Reply:
dont' alias to <created/modified>Meta. keep it `createdAt` `modifiedAt`

Reply:
ok